src/scene/scene_phongmaterial.js

```diff
@@ -210,6 +210,7 @@ pc.extend(pc, function () {
             this.specularMapTint = false;
             this.emissiveMapTint = false;
             this.emissiveIntensity = 1;
+            this.mask = 1;

```

Comment:
Maybe, but shader generation and light usage in shader depends on shader generator implementation.